spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -414,6 +415,49 @@ def enable_educational_discount
         })
       end
 
+      #
+      # Review Submissions
+      #
+
+      def get_ready_review_submission(client: nil, platform: nil, includes: nil)
+        client ||= Spaceship::ConnectAPI
+        platform ||= Spaceship::ConnectAPI::Platform::IOS
```

Comment:
```suggestion
      def get_ready_review_submission(client: nil, platform:, includes: nil)
        client ||= Spaceship::ConnectAPI
```

Thoughts on requiring platform and removing the default? It got me in trouble and confused when testing this on a Mac app 😛 I know we do this in other places in spaceship but I think I'd like to get away from this if possible 🤷‍♂️

Reply:
I've accepted all your suggestions for mandatory fields! 😀